casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -69,6 +74,9 @@ sealed abstract class MultiParentCasperInstances {
       F[_]: Monad: Capture: NodeDiscovery: TransportLayer: Log: Time: ErrorHandler: SafetyOracle](
       storageLocation: Path,
       storageSize: Long,
+      pk: Array[Byte],
```

Comment:
`pk` stands for `publicKey` ? it could also be `privateKey`. I know, existence of `sk` below makes for a good heuristic but could you use full name?

Reply:
If it's standard then I'm fine. Don't want to be to trying. Let's leave it this way.

Reply:
I think if @goral09  didn't immediately understand it then we should use "publicKey" and "secretKey". But I don't think it is a blocker either.